src/pages/index.js

```diff
@@ -83,10 +219,62 @@ addNewCardButton.addEventListener("click", () => {
   addCardPopup.open();
 });
 
-// Form validation setup
-document
-  .querySelectorAll(validationConfig.formSelector)
-  .forEach((formElement) => {
-    const formValidator = new FormValidator(validationConfig, formElement);
-    formValidator.enableValidation();
-  });
+document.querySelectorAll(validationConfig.formSelector).forEach((formElement) => {
+  const formValidator = new FormValidator(validationConfig, formElement);
+  formValidator.enableValidation();
+});
+
+const syncInitialCards = async () => {
+  if (isInitialCardsSynced()) {
+    console.log("Initial cards are already synced");
+    return;
+  }
+
+  try {
+    const serverCards = await api.getInitialCards();
+    const serverCardIds = serverCards.map((card) => card._id);
+    const cardsToAdd = initialCards.filter((card) => !serverCardIds.includes(card._id));
+
+    if (cardsToAdd.length > 0) {
+      const addCardPromises = cardsToAdd.map((card) => {
+        console.log(`Adding card: ${card.name}`);
+        return api.addCard({
+          name: card.name,
+          link: card.link,
+        });
+      });
+
+      await Promise.all(addCardPromises);
```

Comment:
Please, delete it. You get the cards and you don't need to **post` them again to the server. You need to render them on the page. You do it in line 259